makoctl.1.scd

```diff
@@ -25,26 +25,43 @@ Sends IPC commands to the running mako daemon via dbus.
 	*-g, --group*
 		Dismiss the first notification group.
 
+	*-n* <id>
+	    Dismiss the notification with the given id. Defaults to the first
+		notification.
+
 *restore*
 	Restores the most recently expired notification from the history buffer.
 
-*invoke* [action]
-	Invokes an action on the first notification. If _action_ is not specified,
-	invokes the default action.
+*invoke* [-n <id>] [action]
+	Invokes an action on a notification. If _action_ is not specified, invokes
+	the default action. Action names can be discovered using `makoctl list`.
+
+	Options:
+
+	*-n* <id>
+	    Invoke the action on the notification with the given id. Defaults to
```

Comment:
This is using spaces instead of tabs, and makes the build fail I believe.

Reply:
Indeed, I wonder what plugin failed to only use spaces in those three spots… sigh.